packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -71,13 +71,14 @@ function SingleImageViewer({
         overflow='hidden'
         width='100%'
       >
-        {title?.id && title?.text && (
-          <title id={title.id}>{title.text}</title>
-        )}
         <svg
           style={{ touchAction: 'none' }}
           viewBox={`0 0 ${naturalWidth} ${naturalHeight}`}
+          aria-labelledby={title?.id}
```

Comment:
In theory, the title of an SVG is the first `<title>` immediately after the `<svg>` tag. In practice, you have to explicitly label embedded SVGs to ensure that browser HTML parsers detect the title.